src/bp-core/compatibility/class-bb-wpml-helpers.php

```diff
@@ -224,6 +227,56 @@ public function bb_wpml_member_profile_topic_reply( $r ) {
 			return $r;
 		}
 
+		/**
+		 * Remove WPML post__in filter to allow parent translated post as well if the post is not translateable.
+		 *
+		 * @since BuddyBoss [BBVERSION]
+		 * 
+		 * @param string $q Query for parsing WP QUERY.
+		 * 
+		 * @return $q Returns modifed Query.
+		 */
+		public function remove_wpml_post_parse_query( $q ) {
+			if ( isset( $q->query_vars['post__in'] ) ) {
+				unset( $q->query_vars['post__in'] );
+			}
+
+			return $q;
+		}
```

Comment:
```suggestion
		/**
		 * Remove WPML post__in filter to allow parent translated post as well if the post is not translatable.
		 *
		 * @since BuddyBoss [BBVERSION]
		 *
		 * @param WP_Query $q Query for parsing WP QUERY.
		 *
		 * @return WP_Query $q Returns modified Query.
		 */
		public function bb_remove_wpml_post_parse_query( $q ) {
			if ( isset( $q->query_vars['post__in'] ) ) {
				unset( $q->query_vars['post__in'] );
			}

			return $q;
		}
```